### PR TITLE
Trigger builder-base update for go 1.21.11, 1.22.4

### PR DESCRIPTION
**Pull request by sp1999**
*Issue #, if available:*

*Description of changes:*
#1432 and #1435

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: eks-distro-base/make-tests/expected/standard/images/no-deps-presubmit [hunk 1]
```diff
@@ -1,8 +1,8 @@
 
 # This file was generated by running `make run-make-tests` and `make update-make-tests-expected`. DO NOT EDIT
 
-/override/root/../scripts/buildkit_check.sh
+/Users/sjparekh/Documents/local_development/github/eks-distro-build-tooling/eks-distro-base/../scripts/buildkit_check.sh
```
Comment (bhavi-koduru) [inline eks-distro-base/make-tests/expected/standard/images/no-deps-presubmit:4]: this is pointing to your local path, could you check this?

Comment (sp1999) [reply eks-distro-base/make-tests/expected/standard/images/no-deps-presubmit]: Updated

**Pull request merged**